### ISSUE TITLE
Terraform console crash on 0.11.13

### ISSUE THREAD

**Issue by jamengual** [hashicorp/terraform#20943]
### Terraform Version
```
Terraform v0.11.13
```

### Terraform Configuration Files
```
crash happens with or without config files
```

### Crash Output
https://gist.github.com/jamengual/301e0a44a74d5961f8bb902a52549f72

### Expected Behavior
it should lookup the resource

### Actual Behavior
Crashed

### Steps to Reproduce
Please list the full steps required to reproduce the issue, for example:
1. `terraform console`
2. `data.aws_vpc_peering_connection.prod.id`

### Additional Context
MacOs Mojave

### References
Some similar but not quite the same error

**Comment by pselle** [comment]
Hi @jamengual! Sorry to hear you're having trouble. It would help to know a bit more about your Terraform configuration (ex. the basics of your `main.tf` file), or some of those issues you think are similar, but not quite the same, to be able to reproduce this issue.

Looking at the crash report, this line: `2019/04/05 12:26:46 [INFO] command: backend initialized: <nil>` looks suspicious to me, and one idea is that something is going on with your Terraform install itself, if you wanted to give that a look to make sure it's otherwise healthy.

**Comment by jamengual** [comment]
Hi.

I just run the console right after I pushed a vpc live with terraform apply and I just wanted to try the console and this happened so it seems that the rest of the commands work just fine.

I was running version 0.11.12 so I tried to upgrade to see if the error goes away but it did not.

no matter what lockup data. resource command I run, it crashes.

**Comment by hawk0013** [comment]
Take a look here - the workaround might work for you. It did for me.
https://github.com/hashicorp/terraform/issues/20712

**Comment by apparentlymart** [comment]
Hi @jamengual!

This particular crash looks familiar and I think it's one we found and fixed in a later version of Terraform. If it's indeed the same root cause I'm thinking of, the issue was that `terraform console` was running expressions in a context where not all of the referenceable objects were properly populated, and so accessing some would crash.

While I know this is not a very satisfying answer, I suspect if you upgrade to the latest version of Terraform you'll find this issue fixed. If not, it will hopefully at least give a better error message because the functionality for resolving expressions was reworked in Terraform 0.12 to improve its robustness.

**Comment by jamengual** [comment]
no problem , this is solved in 0.12 but I can't upgrade too  it yet.

you can close it.

**Comment by teamterraform** [comment]
Thanks for confirming, @jamengual!
